src/test/java/com/wikia/webdriver/testcases/mercurytests/LightboxTests.java

```diff
@@ -1,138 +1,161 @@
+    lightbox.clickGalleryImage(0);
+    Assertion.assertTrue(lightbox.isDifferentImageAfterSwiping(touchAction,
+                                                               PerformTouchAction.DIRECTION_LEFT,
+                                                               10), "Swiping to left doesn't work");
+    Assertion.assertTrue(
+        lightbox.isDifferentImageAfterSwiping(touchAction, PerformTouchAction.DIRECTION_RIGHT, 10),
+        "Swiping to right doesn't work");
   }
 
   // MT04
   @Test(groups = {"MercuryLightboxTests_004", "MercuryLightboxTests", "Mercury"})
   public void MercuryLightboxTests_004_ZoomThroughPanninAndDoubleTapping() {
-    MercuryBasePageObject base = new MercuryBasePageObject(driver);
-    MercuryArticlePageObject article =
-        base.openMercuryArticleByName(wikiURL, MercuryContent.MERCURY_GALLERY_TEST_TWO);
+    BasePageObject base = new BasePageObject(driver);
+    base.openMercuryArticleByName(wikiURL, MercuryArticles.MERCURY_GALLERY_TEST_TWO);
     PerformTouchAction touchAction = new PerformTouchAction(driver);
-    LightBoxMercuryComponentObject lightbox =
-        article.clickGalleryImage(MercuryContent.MERCURY_GALLERY_IMAGE_INDEX);
-    lightbox.verifyCurrentImageIsVisible();
-    lightbox
-        .verifyZoomingByGesture(touchAction, LightBoxMercuryComponentObject.ZOOM_METHOD_GESTURE);
+    LightboxComponentObject lightbox = new LightboxComponentObject(driver);
+    lightbox.clickGalleryImage(0);
+    Assertion.assertTrue(lightbox.isZoomingByGestureWorking(touchAction, ZOOM_METHOD_GESTURE),
+                         "Zoom by gesture doesn't work");
     lightbox.clickCloseButton();
-    lightbox = article.clickGalleryImage(MercuryContent.MERCURY_GALLERY_IMAGE_INDEX);
-    lightbox.verifyCurrentImageIsVisible();
-    lightbox.verifyZoomingByGesture(touchAction, LightBoxMercuryComponentObject.ZOOM_METHOD_TAP);
+    lightbox.clickGalleryImage(0);
+    Assertion.assertTrue(lightbox.isZoomingByGestureWorking(touchAction, ZOOM_METHOD_TAP),
+                         "Zoom by tap doesn't work");
   }
 
   // MT05
   @Test(groups = {"MercuryLightboxTests_005", "MercuryLightboxTests", "Mercury"})
   public void MercuryLightboxTests_005_TapOnCenterShowHideUI() {
-    MercuryBasePageObject base = new MercuryBasePageObject(driver);
-    MercuryArticlePageObject article =
-        base.openMercuryArticleByName(wikiURL, MercuryContent.MERCURY_GALLERY_TEST_TWO);
+    BasePageObject base = new BasePageObject(driver);
+    base.openMercuryArticleByName(wikiURL, MercuryArticles.MERCURY_GALLERY_TEST_TWO);
     PerformTouchAction touchAction = new PerformTouchAction(driver);
-    LightBoxMercuryComponentObject lightbox =
-        article.clickGalleryImage(MercuryContent.MERCURY_GALLERY_IMAGE_INDEX);
-    lightbox.verifyCurrentImageIsVisible();
-    lightbox.verifyVisibilityUI(touchAction);
+    LightboxComponentObject lightbox = new LightboxComponentObject(driver);
+    lightbox.clickGalleryImage(0);
```

Comment:
What do You think about separate this image index (0) to constance ? You use it across few methods.